peer/brontide.go

```diff
@@ -75,6 +75,10 @@ const (
 
 	// ErrorBufferSize is the number of historic peer errors that we store.
 	ErrorBufferSize = 10
+
+	// peerStorageDelay is the required for a peer to stay before acking a
+	// YourPeerStorageMessage. This required to reduce spamming.
+	peerStorageDelay = 2 * time.Second
```

Comment:
Please remove this for the time being. This is a security related implementation detail that may or may not be the approach we want to go with. Don't commit to this approach at this stage.

Reply:
It is just that it was part of the bolt document